packages/proxy/lib/http/request-middleware.ts

```diff
@@ -233,6 +248,7 @@ const SendRequestOutgoing: RequestMiddleware = function () {
 export default {
   LogRequest,
   ExtractIsAUTFrameHeader,
+  MaybeSimulateSecHeaders,
```

Comment:
It's a Friday...where is this being used? It looks like we are only referencing this in the tests

Reply:
The middleware is applied in the `Http` handler, which executes the middleware in the order they are defined.  The tests reference this directly, which checks the export order of the middleware.

Reply:
OH ok! thanks 😄